### DISCUSSION TITLE
I don't own a phone, and I never will.

### DISCUSSION THREAD

**Discussion by Devin-R-Towell** [community/community#140202] [Other Feature Feedback, Questions, & Ideas]
Hey guys this is not fair, not all of us have a cell phone or access to one. If you force two way authentication in order to log in you will effectively block a lot of people from being able to code or use your website. 
Not everyone can afford a cell phone here in the United States.
I life in an area of Washington were the cheapest cell phone cost 500$ a month. With the sever increasing cost of rent or housing, not all of use can afford to get both groceries pay rent and pay a phone bill that expensive. Without having to choose to either go homeless or not eat for the month. Please don't do this to us. I rely on this website to keep backups of my code, keep my code open source, and share project files. There has got to be another way, can't you do an email authentication alternative to those of us without cell phones?

**Comment by queenofcorgis** [comment]
Hey there! We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.
